app/models/ncr/work_order.rb

```diff
@@ -171,8 +177,12 @@ def final_approver
       end
     end
 
-    def email_approvers
-      Dispatcher.on_proposal_update(self.proposal, self.modifier)
+    def budget_approvals
+      self.individual_approvals.offset(1)
```

Comment:
yes, fine to merge as-is.

Only outstanding comment is https://github.com/18F/C2/pull/634/files#r44310834 afaik @afeld